src/styles/_section_course_curriculum.scss

```diff
@@ -1,97 +1,284 @@
 // Course Curriculum
 // ======================================
 
-.course-curriculum {
-  @include section-height-preset($section-height__medium);
+[class*="course-curriculum"] {
 
   .section__heading {
     margin: 0 0 $gutter-width * 1.5;
   }
 }
 
-.course-curriculum__chapter-list {
-  list-style: none;
-}
+.course-curriculum {
+  @include section-height-preset($section-height__large);
```

Comment:
what kind of value does section-height-preset except? Is that a number? Maybe it could be a string, like 'large', or 'medium'. Not sure what's best, seems redundant to have the mixin name then the variable w/ the same name, I don't know anything about the mixin though

Reply:
Resolved in: https://github.com/thinkific/fundamentals/commit/ade834138676d2b84171f3ad7fccf5933e5e39d3

Reply:
Details will be in the WIKI. Basically, the new "section-height()" mixin accepts a number with unit and the reformed section-height-preset accepts the string of 'small', 'medium' and 'large'. So the include on a section would be @include section-height(section-height-preset(large));